.gitpod.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM gitpod/workspace-full
```

Comment:
Recommends using debian https://github.com/RXT0112/Zernit/blob/init-new/gitpod/debian.Dockerfile it's way more stable, secure, maintained by competent downstream and does not require manual export of binaries

See https://github.com/RXT0112/Zernit/blob/init-new/gitpod/debian.Dockerfile

Reply:
Counterpoint, we maintain the Gitpod images and they are optimized for Gitpod let's stick with the current one